src/io/openshift/plugins/analytics.groovy

```diff
@@ -5,6 +5,37 @@ import io.openshift.Events
 def register() {
     Events.on("build.pass") {
         e, a -> echo "invoking bayesian analytics $e $a"
+        def image = config.runtime() ?: 'node'
+        spawn (image: image, version: config.version()) {
+          echo "Pod spawned."
+          def url = Utils.shWithOutput(this, "git config remote.origin.url")
+          echo "URL:$url"
+          image = config.runtime() ?: 'maven'
```

Comment:
why are assigning the image value again, and we are defaulting it to maven and doing nothing if we have an maven image, any specific reason for it ?